app/models/merge_request_diff.rb

```diff
@@ -47,6 +47,23 @@ def diffs
     @diffs ||= (load_diffs(st_diffs) || [])
   end
 
+  def diffs_no_whitespace
+    # Update ref for merge request
+    merge_request.fetch_ref
```

Comment:
@yms9654 I think what @stanhu is trying to say is that there is no need to do `fetch_ref ` since it will be done by GitLab anyway on new push. So this action is redundant